mantidimaging/gui/windows/main/presenter.py

```diff
@@ -524,12 +524,15 @@ def add_recon_item_to_tree_view(self, parent_id: uuid.UUID, child_id: uuid.UUID,
         :param recon_count: The number of the recon in the dataset. One indicates the first recon that has been added.
         """
         dataset_item = self.view.get_dataset_tree_view_item(parent_id)
-        if recon_count == 1:
+        recon_group = None if recon_count == 1 else self.view.get_recon_group(dataset_item)
```

Comment:
Could we simplify this by not passing `recon_count` as an argument, and just checking if there is an existing recon_group?

I think we could also avoid generating the name here, and use the stack name.
`self.model.get_images_by_uuid(child_id).name`
(I don't think it matters that name from the stack has an underscore in it).